docs/sources/flow/tutorials/assets/generate.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
not super important, but you may want to use `#!/usr/bin/env bash` for better portability and consistency with other scripts in this repo